include/scrnintstr.h

```diff
@@ -620,6 +620,9 @@ typedef struct _Screen {
     SetScreenPixmapProcPtr SetScreenPixmap;
     NameWindowPixmapProcPtr NameWindowPixmap;
 
+    /* This field is used by the 470 proprietary nvidia DDX driver, and should always be NULL */
+    PixmapPtr pScratchPixmap;   /* scratch pixmap "pool" (unused / NULL in modern servers) */
```

Comment:
maybe give it different name that's making clear this is a "reserved" field for nvidia driver ?